### DISCUSSION TITLE
Unraid Docker Template - Multiple Network Option

### DISCUSSION THREAD

**Discussion by VanGoghComplex** [blakeblackshear/frigate#10797] [General]
I'm sorry if this is in the wrong place; I'm new to this and don't know if this applies specifically to the template I get in Unraid or if this is common to all Docker installs.

I've got my camera network segregated from my home network for security. Frigate does well at interfacing with both networks (pulling RTSP streams from the cam net connected to my managed switch via a dedicated host NIC, serving the web interface on my home net), but connecting it to the second network required a post argument `docker network connect` command.

This is doable, but a recurring bug in Unraid prevents the container from restarting correctly after a stop of the Docker service (either manually or because of a reboot / power outage) due to the way the secondary network is handled.

Most of the places I've seen talking about setting home home NVR say segregating your networks is best practice, so I think the template should natively support connection to a second network as an option.

Thanks for your time!

**Comment by NickM-27** [comment]
the template in unraid is not maintained by this repo or the frigate devs. You can do this by adding a manual arg in the extra args section that adds the second network

**Reply by VanGoghComplex** [reply]
Thank you - I've been doing that already, but as stated, Unraid has an intermitted bug that causes issues with that method.

I'll see if I can track down who maintains the Unraid template. Appreciate it!

**Reply by NickM-27** [reply]
if unraid has a bug then changing the template won't fix anything. The template still has to adhere to the actual docker command structure (the same that you have access to)

**Reply by VanGoghComplex** [reply]
I think in this instance it's a problem with the way Unraid handles post arguments vs the way it handles actual run commands. It doesn't run the post args every time it spins up the container, just on initial creation, so the network ID assigned when you run `docker network connect` in the post args is no longer valid after the docker service is restarted following a reboot.

**Reply by NickM-27** [reply]
I wasn't suggesting you use post arguments, I was suggesting using extra arguments to make it part of both networks
